src/data-table/column-numerical.js

```diff
@@ -33,7 +33,7 @@ type NumericalOperations =
 
 type OptionsT = {|
   filterable?: boolean,
-  format?: NumericalFormats,
+  format?: NumericalFormats | ((value: number) => string | number),
```

Comment:
What's the use-case for the result to be a number?

```suggestion
  format?: NumericalFormats | ((value: number) => string),
```

Reply:
The reason behind keeping it as a number was that a user can apply some mathematical transformation which is not supported by NUMERICAL_FORMATS, example Math.ceil.